test/tests/ver-1ia/ver-1ia.i

```diff
@@ -0,0 +1,123 @@
+k_b = '${units 1.380649e-23 J/K}' # Boltzmann constant (from PhysicalConstants.h - https://physics.nist.gov/cgi-bin/cuu/Value?r)
```

Comment:
Same comment about A and B vs. H and D, please correct it everywhere using A and B to be consistent.